generate-schema.js

```diff
@@ -1,10 +1,10 @@
 let fs = require('fs')
 
 // The template for each distribution. Typically, you should not need to modify this.
-const system_template = function(os, distribution) {
+const system_template = function(distribution) {
     return {
         "properties": {
-            "os": { "const": os },
+            "os": { "type": "string" },
```

Comment:
It would be nice to keep this as an enum for validation. I guess there's not a way to tell if a system's OS is Linux right now, but what if we added `"os": "linux"` to `systems.json` (like with the new Windows entry), and then passed the OS in here? Or just pass the whole `system` object here to get `system.os` and `system.distribution`.